.github/workflows/build.yml

```diff
@@ -1,27 +1,28 @@
 name: Build libraries
+
 on:
-    workflow_dispatch:
-    pull_request:
-        types: [opened, synchronize]
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize]
 
 jobs:
-    build:
-        name: Build libraries
-        runs-on: macos-latest
-        steps:
-            -   name: Checkout repository
-                uses: actions/checkout@v2
-                with:
-                    fetch-depth: 0
+  build:
+    name: Build libraries
+    runs-on: macos-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
 
-            -   name: Setup Gradle
-                uses: gradle/gradle-build-action@v2
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
 
-            -   name: Check
-                run: ./gradlew detekt
+      - name: Check
+        run: ./gradlew detekt
 
-            -   name: Build project
-                run: ./gradlew build
+      - name: Build project
+        run: ./gradlew build
```

Comment:
Yes, because currently, we don't have test. Firsts test will be added in https://github.com/allegro/json-logic-kmp/pull/5 and in that PR should be changed build workflow.

Reply:
Ok, so don't forget about adding it :P